spec/acceptance/suites/default/05_install_tpms_spec.rb

```diff
@@ -7,16 +7,16 @@
   def download_rpm_tarball_on(hirs_host, rpm_staging_dir)
     if hirs_host.host_hash[:roles].include?('tpm_2_0')
       tpm_rpms_tarball_url_string = ENV['BEAKER_tpm_2_0_rpms_tarball_url'] || \
-      'https://github.com/op-ct/simp-tpm2-rpms/releases/download/0.1.0-rpms/simp-tpm2-simulator-1119.0.0-0.el7.centos.x86_64.rpm'
+      'https://packagecloud.io/simp-project/6_X_Dependencies/packages/el/7/simp-tpm2-simulator-1332.0.0-0.el7.x86_64.rpm'
     else 
       os = fact_on(hirs_host,'operatingsystemmajrelease')
       tpm_rpms_tarball_url_string = ENV['BEAKER_tpm_1_2_rpms_tarball_url'] || \
-      "https://github.com/m-morrone/simp-tpm12-rpms/releases/download/v0.1-beta/simp-tpm12-simulator-4769.0.0-0.el#{os}.x86_64.rpm"
+      "https://packagecloud.io/simp-project/6_X_Dependencies/packages/el/#{os}/simp-tpm12-simulator-4769.0.0-0.el#{os}.x86_64.rpm"
```

Comment:
Once the packages are published, we should get rid of this and just throw the repos into the nodesets and use `install_package_on`.

Reply:
Updated tests to pull rpms from packagecloud.